ee/clickhouse/sql/cohort.py

```diff
@@ -1,3 +1,18 @@
 CALCULATE_COHORT_PEOPLE_SQL = """
 SELECT distinct_id FROM ({latest_distinct_id_sql}) where {query} AND team_id = %(team_id)s
 """
+
+CREATE_COHORTPEOPLE_TABLE_SQL = """
+CREATE TABLE cohortpeople
+(
+    person_id UUID,
+    cohort_id Int64,
+    team_id Int64,
+    sign Int8
+) ENGINE = CollapsingMergeTree(sign)
```

Comment:
Problem - this needs to be `replicated` as the events/person tables are.

From ee/clickhouse/sql/clickhouse.py:
```
TABLE_ENGINE = (
    "ReplicatedReplacingMergeTree('/clickhouse/tables/{{shard}}/posthog.{table}', '{{replica}}', {ver})"
    if CLICKHOUSE_REPLICATION
    else "ReplacingMergeTree({ver})"
)
```

Reply:
yup good point

Reply:
I added storage policy too assuming it's the same as the other tables